### DISCUSSION TITLE
Feature Request: Ability to use the alert summary as title for the Microsoft Teams notification

### DISCUSSION THREAD

**Discussion by sdaus** [grafana/grafana#66098] [Alerting]
Currently, configuring the summary of an alert as the title for the Microsoft Teams notification in Grafana Alerts requires users to write a custom title with Go, which can be a complex and time-consuming process. To simplify this, it would be great to be able to retrieve specific information from the alert (such as the summary) using something like an environment variable or some other user-friendly method.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82759. Feel free to continue the discussion around this there. Thank you!
